qa/integration/specs/es_output_how_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../framework/fixture'
+require_relative '../framework/settings'
+require_relative '../services/logstash'
+
+describe "a config which indexes data into Elasticsearch", :integration => true do
+
+  before(:all) {
+    @fixture = Fixture.new(__FILE__)
+  }
+
+  after(:all) {
+    es_client = @fixture.get_service("elasticsearch").get_client
+    es_client.indices.delete(index: 'logstash-*')
+    @fixture.teardown
+  }
+
+  it "can ingest 300K log lines" do
+    logstash_service = @fixture.get_service("logstash")
+    es_service = @fixture.get_service("elasticsearch")
+    puts "Ingesting 300K lines of apache logs to ES. This may make your CPU sing.."
```

Comment:
Actually it doesn make my cpu sing, spotify keeps breaking when this script is running.